dcc-portal-ui/app/scripts/stackedbarchart/js/directives.js

```diff
@@ -56,9 +56,19 @@
             label: $scope.yLabel,
             ticks: 4
           },
-          onClick: function(link){
+          onClick: function(data){
             $scope.$emit('tooltip::hide');
-            $location.path(link).search({});
+            if(_.contains(Page.page(), 'projects')){
+              $location.path(data.link).search({});
+            } else {
+              let filter = {file: {projectCode: {is: [data.name]}, primarySite: {is: [data.key]}}};
+              if(_.contains(Page.page(), 'aws')){
```

Comment:
Since AWS and COLLAB are basically the same page and sharing the directive, I can check for its context and send the `repoName` filter value through directive attribute. It would be like moving the `if else` condition from stackedbarchart directive to repositories controller.